backend/src/LibExecution/DvalReprInternalRoundtrippable.fs

```diff
@@ -43,7 +41,7 @@ module FormatV0 =
     | DObj of DvalMap
     | DError of DvalSource * string
     | DIncomplete of DvalSource
-    | DHttpResponse of DHTTP
+    | DHttpResponse of (int64 * List<string * string> * Dval)
```

Comment:
FYI: including the parens here means something different, so you can remove them.